<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="map_picture_line">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>map</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>sat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>sat.skl</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="adress_edit"/>
    </item>
    <item>
     <widget class="QPushButton" name="search_button">
      <property name="text">
       <string>search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_post_index">
      <property name="text">
       <string>post index</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clean_button">
      <property name="text">
       <string>clean</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
